masschallenge/django-accelerator
# [AC-8363] Add birth_year field to core profile

## [AC-8363](https://masschallenge.atlassian.net/browse/AC-8363)

**Changes introduced**
- add birth_year field to core profile

**Testing**
- checkout to this branch
- After [AC-7802](https://github.com/masschallenge/accelerate/pull/2800), running `make migrate` shows migrations have run successfully, however, it does not update the db tables. As an alternative, you can apply migrations using the impact API container. Running migration using accelerate container does not work.
   - on `impact-api/docker-compose.yml`. comment out `assets`, `front-end`, `start_dependencies` definition. Also remove `assets` and `front-end` from `web:... depends_on:` (optional - I've mentioned this just in case you need to save some build time)
   - To apply migrations: `docker-compose run --rm web ./manage.py migrate accelerator`
- Confirm that:
  - `birth_year` exists on `EntrepreneurProfile`, `ExpertProfile` and `MemberProfile` models
  - core_profile_factory to includes `birth_year` (default to null)
  - `birth_year` is a DateField
  - `birth_year` not required
